etc/profile-a-l/cawbird.profile

```diff
@@ -0,0 +1,45 @@
+# Firejail profile for cawbird
+# Description: Open-source Twitter client for Linux
+# This file is overwritten after every install/update
+# Persistent local customizations
+include cawbird.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/cawbird
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-programs.inc
+include disable-passwdmgr.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+apparmor
+caps.drop all
+machine-id
+netfilter
+nodvd
+no3d
+nogroups
+nonewprivs
+noroot
+nosound
+notv
+nou2f
+novideo
+protocol unix,inet,inet6
+seccomp
+shell none
+tracelog
+
+disable-mnt
+private-bin cawbird
+private-cache
+private-dev
+private-tmp
```

Comment:
Can we add `private-etc`?

Reply:
It works fine without. If we add `private-etc` the only two things it doesn't work without are: `ssl` and `resolv.conf` on my network.

Reply:
@kortewegdevries let's do it!

Reply:
This? `alternatives,ca-certificates,crypto-policies,fonts,gtk-2.0,gtk-3.0,hosts,host.conf,hostname,mime.types,nsswitch.conf,pki,resolv.conf,ssl,xdg,X11`

Reply:
Seems right!

Reply:
Done, thanks!